packages/integration-tests/index.js

```diff
@@ -676,20 +677,16 @@ async function buildWorkflow(
   meta
 ) {
   setProcessEnvironment(stackName, bucketName);
+
   const template = await getWorkflowTemplate(stackName, bucketName, workflowName);
-  let collectionInfo = {};
-  if (collection) {
-    collectionInfo = await new Collection()
-      .get({ name: collection.name, version: collection.version });
-  }
-  let providerInfo = {};
-  if (provider) {
-    providerInfo = await new Provider().get({ id: provider.id });
-  }
+  const collectionInfo = collection ? await new Collection().get(collection) : {};
```

Comment:
I actually don't know the answer to this, but would `.get` complain if `collection` was a full collection object with more than `name` and `version`? Or do we always know that it only contains name/version?

Reply:
Good to know